src/transaction/log_impl.h

```diff
@@ -620,7 +620,8 @@ struct log_global
   LOG_LSA flushed_lsa_lower_bound;	/* lsa */
   pthread_mutex_t chkpt_lsa_lock;
 #endif				/* SERVER_MODE */
-  LOG_LSA chkpt_redo_lsa;
+  LOG_LSA chkpt_redo_lsa;	/* checkpoint redo lsa working variable; it's the variable
```

Comment:
I am not sure `chkpt_redo_lsa` should be removed, even if it is a working variable. If you're not sure either, please remove the `todo`.